Formula/katana.rb

```diff
@@ -0,0 +1,19 @@
+class Katana < Formula
+  desc "Crawling and spidering framework"
+  homepage "https://github.com/projectdiscovery/katana"
+  url "https://github.com/projectdiscovery/katana/archive/refs/tags/v1.0.3.tar.gz"
+  sha256 "404a3c4cb073c9a30835c5b7e8eed091dd6472e572e231a21959e3971fdc7bc0"
+  license "MIT"
+  head "https://github.com/projectdiscovery/katana.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/katana"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/katana -version 2>&1")
+    assert_match "Started standard crawling", shell_output("#{bin}/katana -u 127.0.0.1 2>&1")
```

Comment:
you can also do `--timeout 1` default is 10 (so that it would fail fast)

Reply:
We don't want it to fail. It was just having issues matching without adding `2>&1`.